dav1d-sys/build.rs

```diff
@@ -34,11 +34,21 @@ mod build {
         let release_path = source.join(release_dir);
 
         if !Path::new(&source.join(".git")).exists() {
-            runner!("git", "clone", "--depth", "1", REPO, &source);
+            runner!("git", "clone", REPO, &source);
         } else {
             runner!("git", "-C", source.to_str().unwrap(), "pull");
         }
 
+        runner!("git", "-C", source.to_str().unwrap(), "checkout", "0.9.2");
```

Comment:
it needs a 
```
git fetch --depth 1 origin <newtag> && git checkout FETCH_HEAD
```

and can be folded in the branch above.

Reply:
Ah via `fetch --depth 1` that should actually work. That's what I was missing, thanks!